crates/wasmi/src/module/instantiate/mod.rs

```diff
@@ -57,7 +57,10 @@ impl Module {
         let handle = context.as_context_mut().store.inner.alloc_instance();
         let mut builder = InstanceEntity::build(self);
 
-        context.as_context_mut().store.bump_resource_counts(self)?;
+        #[cfg(feature = "std")]
```

Comment:
Why is this behind `std` feature?

Reply:
It uses `Box` on the existential

Reply:
Ah, for `no_std` builds in `wasmi` we simply make use of Rust's `core` and `alloc` libs. So when we are using something like `Box` we have to manually import it via `use alloc::boxed::Box;` since there is no `no_std` prelude. I know it is weird but that will fix the `no_std` build error. :)